libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteLine.kt

```diff
@@ -1160,6 +1218,23 @@ internal class MapRouteLine(
                 it.addStringProperty(WAYPOINT_PROPERTY_KEY, propValue)
             }
         }
+
+        fun swapProperties(featureA: Feature, featureB: Feature) {
```

Comment:
There's no significance to the names in this context. The don't represent anything so what would you suggest?

Reply:
Maybe `fromFeature` and `toFeature`? 🤔

Reply:
If you look at the implementation both variables are equal so "from" and "to" wouldn't really be accurate descriptions. They were given generic names because really the names have no significance in this case. There's an even trade in properties. They are both a source and a destination. That's why in this case I hesitate to describe the variables in any specific way.